SpatialGDK/Source/SpatialGDK/Public/Interop/AbstractStaticComponentView.h

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
```

Comment:
Was it necessary to abstract the StaticComponentView ? 
Compared to the other mocked classes, it looks like a self-contained piece of code, that could be set up in the desired state for tests by pushing a few dummy worker ops.

Reply:
Unless the point is also to isolate it from the worker SDK, in which case, it make sense.

Reply:
Mostly it's to isolate from the worker SDK for ease in testing. For instance, a test that just needs the StaticComponentView to always respond "true" to HasAuthority doesn't want to actually build the ComponentData for every entity and add them to a real StaticComponentView with the right authority, they just want to have that one function return true.

Reply:
@aleximprobable also raised this in a comment below, I'm open to the implementation details. My approach was to change the implementation to not have the `UAbstractStaticComponentView`. Instead, there is a Mock which derives straight from USpatialStaticComponentView with an Init(...) method which calls the OnAuthorityChange and OnAddComponent appropriate as suggested so the worker SDK interface is contained within the mock class and the test just passes in entity IDs / authority values / etc - does that sound good?